src/index.js

```diff
@@ -0,0 +1,172 @@
+
+// proxy request. UI fuctionality event listeners section
+const getLocation = () => {
+    let location = document.getElementById("cityNameInput").value
+    const url ='http://127.0.0.1:5000/location';
+
+    return axios.get(url, {params:{q:location}})
+    .then((response)=>{
+
+    let latitude = response.data[0].lat;
+    let longitude = response.data[0].lon
+    console.log(longitude)
+    console.log(latitude)
+    return {latitude:latitude, longitude: longitude};
+
+
+    })
+    .catch( (error) => {
+        console.log('error in findLatitudeAndLongitude!');
+      });
+    
+ }
+
+const getWeather = (lat, lon) =>{
+    return axios
+    .get('http://127.0.0.1:5000/weather', {params:{lat:lat,lon:lon}})
+    .then((response)=>{
+       
+        let result = response.data['main']['temp'];
+        return result
+       
+    })
+    .catch((error)=>{
+        console.log(error);
+    }
+
+    )
+}
+
+const getLocationWeather = () =>{
+    let tempVal = document.getElementById("tempValue");
+    return getLocation()
+    .then( result =>{
+             getWeather(result.latitude, result.longitude)
+            .then(
+                (response)=>{
+                  
+                    let farenheit = Math.floor(((response) - 273.15) * 9/5 + 32 )
+                    console.log(farenheit);
+                    tempVal.textContent = farenheit;
+                    return farenheit
+
+                }
+            )
+          
+          
+            
+        }
+    )
+   
+}
+
+
+
+
+
+// states
+const state = {
+    temp: 0,
+    landscape:"🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+    city: "Seattle"
+
+}
+
+// selected elements
+const temp = document.getElementById("tempValue");
+const controlUp = document.getElementById("increaseTempControl");
+const controlDown = document.getElementById("decreaseTempControl");
+const landscape = document.getElementById("landscape");
+const cityName = document.getElementById("headerCityName");
+const cityInput = document.getElementById("cityNameInput");
+const reset = document.getElementById("cityNameReset");
+const skyImage = document.getElementById("sky");
+const sky = document.getElementById("skySelect");
+const weatherGarden = document.getElementById("gardenContent")
+const tempbutton = document.getElementById("currentTempButton")
```

Comment:
👍 nice job naming this variable in a way that describes what the element is. Use camelcase though like tempButton